File: task-03/java/primeno.java
import java.util.Scanner;
class Primeno
{
	public static void main(String arg[])
	{
	int j,c;
               System.out.print("Enter n: ");
	Scanner s=new Scanner(System.in);
	int n=s.nextInt();
               System.out.println("Prime numbers: ");
	for(int i=2;i<=n;i++)
	{
	c=0;
	for(j=1;j<=i;j++)
	{
	   if(i%j==0)
	   {
	        c++;
	   }
	}
	if(c==2)
	       System.out.print(i+"  ");
	}
	}
}

File: task-03/ruby/primeno.rb
print "Enter n: "
n = gets.chomp.to_i

if n >= 2
  for s in 2..n
    prime = true
    for i in 2..(s - 1)
      if s % i == 0
        prime = false
        break
      end
    end
    puts s if prime
  end
end
